### DISCUSSION TITLE
Butterfly HDTV

### DISCUSSION THREAD

**Discussion by Niyasmkkv** [iptv-org/community#690] [Q&A]
**Channel Name:** Butterfly HD TV
**Channel ID:** ButterflyHDTV.ae
**Country:** AE
**Language:** Malayalam
**Website:** https://www.butterflyhdtv.com/
**Stream URL (optional):** https://play.google.com/store/apps/details?id=com.butterflyhdtv.butterflyhd
**Notes (optional):** xxx

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19723
